ui/lib/config-ui/addon/routes/messages/create.js

```diff
@@ -4,5 +4,24 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { hash } from 'ember-concurrency';
 
-export default class MessagesCreateRoute extends Route {}
+export default class MessagesCreateRoute extends Route {
+  @service store;
+
+  queryParams = {
+    authenticated: {
+      refreshModel: true,
+    },
+  };
+
+  model(params) {
+    const { authenticated } = params;
+
+    return hash({
+      message: this.store.createRecord('config-ui/message', { authenticated }),
+      authenticated,
```

Comment:
Would this need to be set on the route model if you can access it from the ED model (message)?